backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -536,6 +537,23 @@ export class UserResolver {
       userEntity.lastName = lastName
     }
 
+    if (alias) {
+      if (alias.length < 5) {
+        throw new LogError('Given alias is too short', alias)
+      }
+      if (alias.length > 20) {
+        throw new LogError('Given alias is too long', alias)
+      }
+      if (!alias.match(/^[0-9A-Za-z]+$/)) {
```

Comment:
whats the meaning of the ending $-character in the exp?
missing allowed - and _ character.

Reply:
$ is end of string